src/App.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import './App.css';
+
+import users from './api/users';
+import todos from './api/todos';
+
+import { TodosList } from './components/todosList';
+
+const TodosWithUsers = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId).name,
+  isVisible: true,
+}));
+
+class App extends React.Component {
+  state = {
+    renderList: [...TodosWithUsers],
+    title: '',
+    user: '',
+    completed: false,
+    taskError: true,
+    userError: true,
+  }
+
+  eventTitleOnChange = event => (
+    this.setState({
+      title: event.target.value,
+      taskError: true,
+    }));
+
+  eventUserOnChange = event => this.setState({
+    user: event.target.value,
+    userError: true,
+  });
+
+  addNewTask = (event) => {
+    event.preventDefault();
+    const { title, user } = this.state;
+
+    if (!title) {
+      this.setState({ taskError: false });
+    } else {
+      this.setState({ taskError: true });
+    }
+
+    if (!user) {
+      this.setState({ userError: false });
+    } else {
+      this.setState({ userError: true });
+    }
```

Comment:
```suggestion
    if (!title) {
      this.setState({ taskError: false });
    }
    
    if (!user) {
      this.setState({ userError: false });
    } 
```
If the condition not passed, the default value is already ```true```.